pages/index.tsx

```diff
@@ -1,12 +1,35 @@
 import type { NextPage } from "next";
-import Link from "next/link";
-import { Container } from "react-bootstrap";
+import { Container, Image } from "react-bootstrap";
+import React from "react";
 
 const Home: NextPage = () => {
   return (
     <Container>
-      <h1>Hello, world!</h1>
-      <Link href="/about">About</Link>
+      <Image
+        src="/blueno.png"
```

Comment:
how can we crop our logo? i would love to have it in our home screen as well as the headers!